examples/e1.20/DEVICE_INFO.json

```diff
@@ -85,15 +85,15 @@
       ]
     },
     {
-      "name": "current_personality",
+      "name": "personality",
       "type": "uint8",
       "ranges": [
         { "minimum": 1, "maximum": 255 }
       ]
     },
     { "name": "personality_count", "type": "uint8" },
     {
-      "name": "dmx_start_address",
+      "name": "dmx_address",
```

Comment:
No real excuse here though, apart from none of us spotted and flagged it in the recent review. Although I'd argue the change is being made the wrong way round, DMX start address is more descriptive and unambiguous to me.

Reply:
I agree I think it should be `dmx_start_address` matching the name of the pid 240. Same goes for the fields name within that parameters schema. I also think wherever personality is used it should be `dmx_personality` matching that parameters name